docs/blogs/kubecon_cfp.md

```diff
@@ -0,0 +1,45 @@
+# ElasticDL: Kubernetes-native Deep Learning System with Elastic Scheduling
+
+## Description
+
+In these years, Kubernetes becomes the fundamental infrastructure of machine
+learning. Kubeflow project provides some operators to run distributed machine
+learning jobs on Kubernetes. It uses gang scheduling policy. For example, a job
+requires N pod, it will keep pending if there are N-1 or less available pods. And
+during execution, the job will crash if any one of these N instances crashes. As
+a result, the job pending time is long and the resouce utilization is insufficient.
+
+ElasticDL implements elastic scheduling policy. A job doesn't need wait for all
+the required resource ready and start promptly. A job process can be preempted
+by a higher priority job, join freely once the resource become sufficient later
+and the entire job keeps running. It can significantly shorten the job waiting
+time and improve the resource utilization.
+
+## Benefits to the Ecosystem
+
+This talk introduces a system ElasticDL, it implements elastic scheduling policy
+which is new to deep learning workloads scheduling on Kubernetes. Compared with
+the gang scheduling policy that kubernetes operators from Kubeflow apply, it
+can improve the resource utilization significantly. We add a master to launch,
+monitor and manage the worker pods using Kubernetes API and distribute the
+sharded data dynamically to them. It doesn't rely on the Kubernetes operators.
+In this way, master can understand not only the internal details of deep
+learning process but also the model structure, and then make more optimized
+scheduling decision.
+
+For system developers and operators, this talk introduces a elastic scheduling
+policy for deep learning jobs to improve the resource utilization.
+
+For model developers, this talk introduces how to simply develop a distributed
```

Comment:
ElasticDL maintains the convergence of learning jobs in addition to providing the above benefits of elastic deep learning.  The mathematical approaches in use are also those used in the community for years, including DistBelief, the prior generation of Google technology before TensorFlow, and Peacock, the system behind Tencent's online advertising system.

Reply:
The audiences of the talk are kubernetes developers / users. Maybe we can ignore the convergence part in the benefit section considering the limit of the character number. Because the audience may be unfamiliar with this part.